### PR TITLE
Group setup

### PR DESCRIPTION
**Pull request by sselin-co**
Wow, a database backup with no merge conflicts!

This PR contains a template tag that queries a users group and returns a boolean depending on group status. It also contains my first unit test so feel free to give me pointers on how it can be improved (though it's a pretty simple function all things considered).

### CODE REVIEW

**Review by Rock-it-science — CHANGES_REQUESTED** [review]
For some reason it looks like this rolls back my test readme changes. I would recommend merging development into this branch to get the recent changes.

**Comment by Rock-it-science** [comment]
Also It looks like I am not getting any top bar with client or employee:
image

**Comment by sselin-co** [comment]
Thanks for catching those issues, when I'm done with work I'll merge development into my branch then push again to hopefully fix the readme.

It looks like you're at the root URL in the image, can you check if it's still a problem on either the employee or client home? I think the issue will be the same regardless, but I want to be sure.

File: LIMS_IMAGE/web/tests/test_accounts_user_tags.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            username='testuser_c', password='asdf')
+        self.test_user_client.groups.add(Group.objects.get(name='Client'))
+        self.test_user_emp = User.objects.create_user(
+            username='testuser_e', password='asdf')
+        self.test_user_emp.groups.add(Group.objects.get(name='Employee'))
+        self.test_user_admin = User.objects.create_user(
+            username='testuser_a', password='asdf')
+        self.test_user_admin.groups.add(Group.objects.get(name='Admin'))
+        self.test_user_anon = User.objects.create_user(
+            username='testuser', password='asdf')
```
Comment (Rock-it-science) [inline LIMS_IMAGE/web/tests/test_accounts_user_tags.py:22]: This is okay, but best practice is to use model bakery with recipes.

File: LIMS_IMAGE/web/tests/test_accounts_user_tags.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        self.assertEqual(self.test_user_emp.groups.filter(
+            name='Employee').exists(), True)
+        self.assertEqual(self.test_user_admin.groups.filter(
+            name='Admin').exists(), True)
+        self.assertEqual(self.test_user_anon.groups.filter(
+            name='Client').exists(), False)
+        self.assertEqual(self.test_user_anon.groups.filter(
+            name='Employee').exists(), False)
+        self.assertEqual(self.test_user_anon.groups.filter(
+            name='Admin').exists(), False)
```
Comment (Rock-it-science) [inline LIMS_IMAGE/web/tests/test_accounts_user_tags.py:36]: I would do a little more than check if the object exists. Again, this is fine, but I would take a look at my other tests for an example of what else we can test.

**Review by Rock-it-science — CHANGES_REQUESTED** [review]

**Comment by Rock-it-science** [comment]
> Thanks for catching those issues, when I'm done with work I'll merge development into my branch then push again to hopefully fix the readme.
> 
> It looks like you're at the root URL in the image, can you check if it's still a problem on either the employee or client home? I think the issue will be the same regardless, but I want to be sure.

image

**Comment by sselin-co** [comment]
> > Thanks for catching those issues, when I'm done with work I'll merge development into my branch then push again to hopefully fix the readme.
> > It looks like you're at the root URL in the image, can you check if it's still a problem on either the employee or client home? I think the issue will be the same regardless, but I want to be sure.
> 
> image

Let me know if you already did this, but did you rebuild the image when pulling my changes? If not, then the changes to the database defining groups hasn't been applied.

File: LIMS_IMAGE/web/tests/test_accounts_user_tags.py (hunk 2 above)
Comment (sselin-co) [reply LIMS_IMAGE/web/tests/test_accounts_user_tags.py:36]: Makes sense, I will push some changes later improving my tests.

**Comment by Rock-it-science** [comment]
> > > Thanks for catching those issues, when I'm done with work I'll merge development into my branch then push again to hopefully fix the readme.
> > > It looks like you're at the root URL in the image, can you check if it's still a problem on either the employee or client home? I think the issue will be the same regardless, but I want to be sure.
> > 
> > 
> > image
> 
> Let me know if you already did this, but did you rebuild the image when pulling my changes? If not, then the changes to the database defining groups hasn't been applied.

Ah, I did not. I restarted the image, but I will try rebuilding it

**Comment by Rock-it-science** [comment]
> > > > Thanks for catching those issues, when I'm done with work I'll merge development into my branch then push again to hopefully fix the readme.
> > > > It looks like you're at the root URL in the image, can you check if it's still a problem on either the employee or client home? I think the issue will be the same regardless, but I want to be sure.
> > > 
> > > 
> > > image
> > 
> > 
> > Let me know if you already did this, but did you rebuild the image when pulling my changes? If not, then the changes to the database defining groups hasn't been applied.
> 
> Ah, I did not. I restarted the image, but I will try rebuilding it

Hey nice, it worked 
image

**Comment by sselin-co** [comment]
I'll make sure to add to our main README with this PR that whenever a database is updated we need to rebuild so we're all on the same page. With that, I will improve my unit tests and fix the testing README.

**Review by shepkeira — APPROVED** [review]

**Review by Rock-it-science — APPROVED** [review]
Looks great! Nice improvements

**Pull request merged**